File: src/main/java/com/camackenzie/exvi/core/VerificationResult.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.camackenzie.exvi.core;

/**
 *
 * @author callum
 */
public class VerificationResult {

    private final int error;
    private final String message;

    public VerificationResult(int err, String msg) {
        this.error = err;
        this.message = msg;
    }

    public int getError() {
        return this.error;
    }

    public String getMessage() {
        return this.message;
    }
}

File: src/commonMain/kotlin/com/camackenzie/exvi/core/api/VerificationResult.kt
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.camackenzie.exvi.core.api

import com.camackenzie.exvi.core.api.DataResult
import com.camackenzie.exvi.core.util.None
import com.camackenzie.exvi.core.util.SelfSerializable
import kotlinx.serialization.json.*
import kotlinx.serialization.*

/**
 *
 * @author callum
 */
@kotlinx.serialization.Serializable
class VerificationResult : DataResult<None> {
    override val result: None? = null

    constructor(err: Int, msg: String) : super(err, msg) {}
}
